test/Interop/Cxx/class/members-typechecker.swift

```diff
@@ -0,0 +1,6 @@
+// RUN: %target-typecheck-verify-swift -I %S/Inputs -enable-cxx-interop
+
+import CxxMembers
+
+var s = MyClass();
+s.const_member = 42; // expected-error {{cannot assign to property: 'const_member' setter is inaccessible}}
```

Comment:
Would be nice to also add a test that prints the generated Swift interface -- see https://github.com/apple/swift/pull/30233 for an example.

Reply:
Done